packages-backend/express/src/auth.spec.js

```diff
@@ -47,3 +47,55 @@ describe('getConfiguredAudience', () => {
     );
   });
 });
+
+describe('writeSessionContext', () => {
+  const mockIssuer = 'https://mc-api.nowhere.com';
+
+  it('should not include anything in request session if no "decoded_token" exists in the request', () => {
+    const request = {};
+
+    writeSessionContext(request);
+
+    expect(request).not.toHaveProperty('session');
+  });
+
+  it('should write "userId" and "projectKey" in request session property', () => {
+    const mockDecodedToken = {
+      iss: mockIssuer,
+      sub: 'user-id-1',
+      [`${mockIssuer}/claims/project_key`]: 'almond-40',
```

Comment:
Let's use a fake project key 😉

Reply:
Updated in this commit: 335c1119c938fd8d183875fd828803fa4afa42d0